tutorial/markdown/nodejs/quickstart-nextjs-capella/quickstart-nextjs-capella.md

```diff
@@ -0,0 +1,804 @@
+
+If the document is updated successfully, we return the document as a JSON response with a 200 status code.
+
+```ts
+/**
+ * @swagger
+ * /api/v1/airline/{airlineId}:
+ *   put:
+ *     summary: Update an airline
+ *     description: |
+ *       Update an airline with specified ID.
+ *
+ *       This provides an example of using [Key Value operations](https://docs.couchbase.com/nodejs-sdk/current/howtos/kv-operations.html) in Couchbase to update a document with specified ID.
+ *
+ *       Key Value operations are unique to Couchbase and provide very high-speed get/set/delete operations.
+ *
+ *       Code: `airline/[airlineId]/route.ts` Method: `PUT`
+ *     tags:
+ *        - Airline
+ *     parameters:
+ *       - name: airlineId
+ *         in: path
+ *         description: ID of the airline
+ *         required: true
+ *         schema:
+ *           type: string
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/Airline'
+ *     responses:
+ *       200:
+ *         description: Returns the updated airline
+ *       400:
+ *         description: Invalid request body
+ *       500:
+ *         description: An error occurred while updating airline
+ */
+export async function PUT(
+  req: NextRequest,
+  { params }: { params: { airlineId: string } }
+) {
+  try {
+    const { airlineId } = params;
+    const airlineData: TAirline = await req.json();
+    const parsedAirlineData = AirlineSchema.parse(airlineData);
+    const { airlineCollection } = await getDatabase();
+
+    const updatedAirline = await airlineCollection.upsert(
+      airlineId,
+      parsedAirlineData
+    );
+    return NextResponse.json(
+      {
+        airlineId: airlineId,
+        airlineData: parsedAirlineData,
+        updatedAirline: updatedAirline,
+      },
+      { status: 200 }
+    );
+  } catch (error) {
+    if (error instanceof ZodError) {
+      return NextResponse.json(
+        { message: "Invalid request body", error: error.errors },
+        { status: 400 }
+      );
+    } else {
+      return NextResponse.json(
+        {
+          message: "An error occurred while updating airline",
+          error: "An error occurred while updating airline",
+        },
+        { status: 500 }
+      );
+    }
+  }
+}
+```
+
+### DELETE Airline
+
+In the `DELETE` method, we use the `remove` method of the `airlineCollection` to delete the document with the specified ID. If the document is not found, we return a 404 status code. If an error occurs, we return a 500 status code.
+
+If the document is deleted successfully, we return a JSON response with a 202 status code.
+
+```ts
+/**
```

Comment:
You can get rid of the Swagger here for the tutorial. It becomes quite lengthy otherwise.

Reply:
removed swagger comments